lib/channel.ts

```diff
@@ -54,9 +54,9 @@ export class Channel {
     this._send(method, params)
   }
 
-  addHandler(method: string, handler: Function) {
+  addHandler(method: string, handler: (...args: any[]) => void) {
```

Comment:
Just a quick question, why can't we use` unknow[]` instead of `any[]`, I might be missing the context here.